lib/log.rb

```diff
@@ -39,6 +39,14 @@ module TBD
   @@tag[ERROR] = "ERROR"
   @@tag[FATAL] = "FATAL"
 
+  @@msg = []
+  @@msg[0]     = ""
+  @@msg[DEBUG] = "For debugging"
+  @@msg[INFO]  = "No errors, no warnings"
+  @@msg[WARN]  = "Non-fatal warnings"
+  @@msg[ERROR] = "Non-fatal errors"
+  @@msg[FATAL] = "Fatal errors"
```

Comment:
Temporary half-sentences for now ...

Reply:
Just a dumb question, do you have to do logging in French and English?

Reply:
Good question (not dumb). Once/if the OS App offers language options (at some point French), I'll rehaul TBD GUI options & I/O messaging/logging. But not in the short term.